src/psnawp_api/models/user.py

```diff
@@ -250,6 +250,75 @@ def is_blocked(self) -> bool:
         ).json()
         return self.account_id in response["blockList"]
 
+    def get_all_trophies(self, limit=799):
+        """get all the trophies info for the user's game
+
+        Args:
+            limit (int, optional): limit of trophies to query. Max is 800. Defaults to 100.
+
+        Returns:
+            dict
+        """
+
+        param = {"limit": min(limit, 800)}
+
+        response = self._request_builder.get(
+            url=f"{User.trophy_base_uri}/v1/users/{self.account_id}/trophyTitles",

```

Comment:
This "user.trophy_base_uri" should be replaced by BASE_PATH['trophy_base_uri'] and BASE_PATH should include the right uri path.

Reply:
fixed that in new commit